### DISCUSSION TITLE
ストックの廃止

### DISCUSSION THREAD

**Discussion by hellowork-mhlw** [increments/qiita-discussions#394] [要望・アイデア・不具合]
**いいね**と**ストック**がありますが、ユーザーにとって分かりにくいので**いいね**だけでいいと思います。 いいねした記事を秘密にしたい場合は、はてブみたいにいいねを非公開に設定できるようにすればいいと思います。
![image](https://user-images.githubusercontent.com/116468267/209469158-f42e1c7a-1247-4154-bb06-23ec159f3069.png)

賛同できる方は👍をして、できない方は👎してください

Keep it simple, stupid.

**Comment by munierujp** [comment]
自分の場合は公開したくないからではなく、あとから参照したい記事を分類するためにストックを使っているので、いいねだけになると困ります。
はてなブックマークなど外部のブックマークサービスでやればいいと言われればそれまでですが……

**Comment by tanaka-asami415** [comment]
@hellowork-mhlw 
ご意見ありがとうございます！

いいねは、記事やコメントを読んで「いい！」「技術的に役立った」と思った時や、
書いた人を「応援したい」という気持ちを表現できる機能で、
ストックは、「あとでじっくり読みたい」「繰り返し読みたい」記事を
保存しておくことができる機能となっております🙇

公開・非公開を分けるものではないので、ストックの廃止は考えておりませんが、
わかりにくいという点は課題として今後の参考にさせていただきます🙇

こちらもよろしければ、ご参照ください。
【関連ヘルプページ】
https://help.qiita.com/ja/articles/qiita-like
https://help.qiita.com/ja/articles/qiita-stocks
